extension/chrome/elements/compose.htm

```diff
@@ -150,31 +150,33 @@ <h1 id="header_title" data-test="header-title">New Secure Message</h1>
         <tr id="password_or_pubkey_container" data-test="password-or-pubkey-container">
           <td>
             <div id="expiration_note">
-              Recipient will have access to this message
+              The recipient will have access to this email
               <a href="#" id="expiration_note_settings_link">
                 for
                 <span id="expiration_note_message_expire">(loading)</span>
               </a>
-              . You are responsible for sharing this password with them (use other medium to share the password - not email).
+              . You are responsible for sharing this password with your recipients (use other mediums to share the password, not email).
               <br />
               <br />
               <span id="password-policy-container"></span>
               <br /><br />
-              (Messages sent to FlowCrypt users don't expire this way and don't need password exchange)
+              (Emails sent to users who have FlowCrypt installed don't expire this way and don't require you communicate a password.)
             </div>
             <div>
-              <div class="warning_nopgp" data-test="warning-nopgp">emails <span class="grey">in grey</span> don't use encryption</div>
-              <div class="warning_revoked" data-test="warning-revoked">emails <span class="orange">in orange</span> have expired or revoked key</div>
+              <div class="warning_nopgp" data-test="warning-nopgp">Emails in <span class="grey">grey</span> don't use encryption.</div>
+              <div class="warning_revoked" data-test="warning-revoked">Emails in <span class="orange">orange</span> have expired or revoked key.</div>
               <br />
-              <a href="#" class="add_pubkey" data-test="action-open-add-pubkey-dialog">Add their Public Key</a>
-              <div class="warning_nopgp">or ask them to get FlowCrypt.</div>
+              Add the recipient's <a href="#" class="add_pubkey" data-test="action-open-add-pubkey-dialog">public key</a>
```

Comment:
there can be multiple recipients, so let's change text to `Add their public keys` and make full text as link (not only `public key`)